services/news/src/views/uc-publisher-ads-req.ejs

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title><%= title %></title>
+  <link rel="stylesheet" href="/css/global.css" />
+  <link rel="icon" href="/img/spy.svg" />
+  <script src="<%= `https://${DSP_HOST}:${EXTERNAL_PORT}/js/dsp.js` %>" dsp="<%= `${DSP_HOST}:${EXTERNAL_PORT}` %>" bucket="<%= `${bucket}` %>" cloudenv="<%= `${cloudEnv}` %>"></script>
```

Comment:
I don't think this is relevant to the current use-case. This script has also been removed from the latest index.ejs on dev branch: https://github.com/privacysandbox/privacy-sandbox-demos/blob/dev/services/news/src/views/index.ejs

Please remove this script and you'll also not need to import the DSP_HOST env var in NEWS.

Reply:
Done.